config/filter.d/nsd.conf

```diff
@@ -22,7 +22,7 @@ _daemon = nsd
 #          (?:::f{4,6}:)?(?P<host>[\w\-.^_]+)
 # Values:  TEXT
 
-failregex =  ^\[\]%(__prefix_line)sinfo: ratelimit block .* query <HOST> TYPE255$
-              ^\[\]%(__prefix_line)sinfo: .* <HOST> refused, no acl matches\.$
+failregex =  ^%(__prefix_line)sinfo: ratelimit block .* query <HOST> TYPE255$
+             ^%(__prefix_line)sinfo: .* <HOST> refused, no acl matches\.$
```

Comment:
I would have to completely remove `^%(__prefix_line)sinfo: .*`, because it makes at all no sense (independent whether greedy or not). Imho, only anchored end after `.*` is important.

But I know some owners, that will immediately beef about this :)))